ttt.py

```diff
@@ -92,7 +91,16 @@ def transcribe_fasterwhisper(calljson, audiofile):
 
     # This whisper wants the path, not bytes but we need to cast it from pathlib to str
     audiofile = str(audiofile)
-    segments, info = model.transcribe(audiofile, beam_size=5, vad_filter=vad_filter)
+    # We are going to set the vad parameters to half a second although env variable still turns
+    # vad off or on globally
+    segments, info = model.transcribe(
+        audiofile,
+        beam_size=5,
+        vad_filter=vad_filter,
+        vad_parameters=dict(min_silence_duration_ms=500),
+        condition_on_previous_text=False,
+        language="en",
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider supporting multiple languages for transcription.

Hardcoding the language to English limits the utility of the transcription service. If possible, consider making the language dynamically selectable based on the audio file or user preference.

```suggestion
        language=language,
```